python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -423,10 +424,8 @@ def last(self, split_every=None, split_out=1):
     def aggregate(self, arg, split_every=None, split_out=1):
         if arg == "size":
             return self.size()
-        arg = _redirect_aggs(arg)
 
-        if not isinstance(arg, dict):
-            arg = {self._slice: arg}
+        arg = _redirect_aggs(arg)
```

Comment:
Haven't checked out the code locally but does this already handle the `not dict` case?

Reply:
Sorry for the delay - was re-familiarizing myself with this branch.

I removed this in an attempt to consolidate the aggregation input normalization entirely within `groupby_agg`, but it looks like using that input normalization would make it so we end up computing the aggregation on all non-groupby columns for series groupbys:

```python
    columns = [c for c in ddf.columns if c not in gb_cols]
    if not isinstance(aggs, dict):
        aggs = {col: aggs for col in columns}
```

Which is unnecessary and inefficient - knowing this, it's best that we add this series groupby specific check back in.